cypress/fixtures/edit-job/single-user-job.json

```diff
@@ -9,7 +9,7 @@
     "enabled": true,
     "leaderOnlyJob": false,
     "externalAccess": false,
-    "jobType": "DATA_INTEGRITY",
+    "jobType": "DISABLE_INACTIVE_USERS",
```

Comment:
I've changed this because the data integrity job registers as modified after loading due to a final form quirk. This caused the test to fail due to a popup, something that didn't happen with the mocked backend (even though it should have). I'll create a follow up issue for addressing the final form quirk.